airgun/entities/syncplan.py

```diff
@@ -12,37 +12,51 @@
 class SyncPlanEntity(BaseEntity):
 
     def create(self, values):
+        """Create new sync plan"""
         view = self.navigate_to(self, 'New')
         view.fill(values)
         view.submit.click()
         view.flash.assert_no_error()
         view.flash.dismiss()
 
     def delete(self, entity_name):
+        """Delete existing sync plan"""
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
         view.actions.fill('Remove')
         view.dialog.confirm()
         view.flash.assert_no_error()
         view.flash.dismiss()
 
     def search(self, value):
+        """Search for sync plan"""
         view = self.navigate_to(self, 'All')
         return view.search(value)
 
     def read(self, entity_name):
+        """Read values for created sync plan"""
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
         return view.read()
 
     def update(self, entity_name, values):
+        """Update sync plan with necessary values"""
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
         filled_values = view.fill(values)
         view.flash.assert_no_error()
         view.flash.dismiss()
         return filled_values
 
+    def add_product(self, entity_name, products_list):
```

Comment:
`add_product` but `products_list`, that's a bit confusing.

Reply:
If no better alternatives - at least add a line in docstring explaining either single product or list is accepted.